rest-api/src/main/scala/pipeline/data/PipelineCluster.scala

```diff
@@ -0,0 +1,58 @@
+package pipeline.data
+
+import java.net.InetSocketAddress
+import java.util.Collections
+
+import com.datastax.driver.core.Cluster
+import com.datastax.driver.core.PreparedStatement
+import com.datastax.driver.core.ResultSet
+import com.typesafe.config.ConfigFactory
+
+import scala.collection.JavaConverters._
+
+object PipelineCluster {
+  val config = ConfigFactory.load()
```

Comment:
The `config` could be passed in argument to your main (and, if none is provided, you could default to `ConfigFactory.load`)